s3/sc-s3-synapse-ra.yaml

```diff
@@ -67,7 +73,7 @@ Resources:
             Sid: "WriteAccess"
             Effect: "Allow"
             Principal:
-              AWS: !Ref 'S3UserARNs'
+              AWS: !Ref 'S3UserARNs' #sc-requestor write access
```

Comment:
you'll need to do the same thing with `WriteAccess` as you did with `ReadAccess` because we want the ARN to have both read/list and write access.

Reply:
This was one open question... Should the Synapse prod account really have write access to this bucket?

Reply:
This was one open question... Should the Synapse prod account really have write access to this bucket?